### ISSUE TITLE
BUG: excalidraw Displaying the Whiteboard,can not use

### ISSUE THREAD

**Issue by smilesxsy** [zsviczian/obsidian-excalidraw-plugin#918]
**Your environment**
Mobile: Black Shark (Xiaomi system) Android System 10 
excalidraw versions： 1.4.19、1.8.1
obsidian versions： 1.3.2、1.4.1
webview: About 88, I forgot the details
**Describe the bug**
You can open the plug-in, but after creating the file you want to write something on the whiteboard

**Comment by smilesxsy** [comment]
pass [bug](https://github.com/zsviczian/obsidian-excalidraw-plugin/issues/181)  I got a new idea to try, and I solved it by changing the webview.
install webview(Android System WebView) 108.0.5359.61 obsidian can not use. After installing 107.0.5304.141, you can open obsidian and use excalidraw.
I have been looking for a solution to this problem for a long time, and I have tried various ways to help you

**Comment by zsviczian** [comment]
Yes, there is a problem with older webviews. I think this has to do with the target build module configured in `tsconfig.json` of the plugin project. The thing is, I am building for `es2017` which seems old enough (and matches the core excalidraw package build environment). Building for any earlier version causes problems with the ExcalidrawAutomate Script Engine execution code.

Your issue with 108.0.5359.61 is strange. That is what I am using as well, and experience no issues with it.

I've had earlier issues with Xiaomi devices and webview, so couldl be something Xiaomi specific.

**Comment by smilesxsy** [comment]
> 

Mobile: Xiaomi Tablet 4 for Android uses 8
excalidraw versions： 1.8.1
obsidian versions： 1.4.1
webview: 71.0.3578.99
I think you may be right. I also have a Xiaomi tablet 4. This problem is solved by upgrading the webview version to 107. The 108 version cannot be installed on the tablet
